website/content/docs/platform/k8s/helm/run.mdx

```diff
@@ -482,6 +482,28 @@ $ kubectl exec -ti <name of pod> -- vault operator unseal
 After a few moments the Vault cluster should elect a new active primary. The Vault
 cluster is now upgraded!
 
+#### Interaction with Raft Autopilot Automated Upgrades
```

Comment:
What prompted adding this to the docs (please could we include that in the PR description)? I'm a little confused, as Nick said it feels a little like this isn't really the way automated upgrades are supposed to run, but I may be missing some context.

I wonder if it would make sense to split this between the two different strategies for managing pods instead of "automated" vs not automated. Am I right thinking that once you've run `helm upgrade`, that's when your options diverge, either a) delete pods 1 by 1 starting with standbys, or b) scale up until autopilot decides the new version is the majority?

Option a) seems essentially the same regardless of whether automated upgrades are available, with the minor detail that the leader will update before you kill it in 1.11+. Option b) is only available from 1.11+ but is materially different, so I think warrants a bit more detail and its own section if we want to include it as an option.

Reply:
I can add more details, but essentially, we had a ticket to document how Raft Autopilot Automated Upgrades would mix with Helm.

When testing out `helm upgrade` with autopilot automated upgrades enabled, I found it very easy to accidentally take down the entire cluster, hence why I wrote this section.

I think what you're saying is that this should be a more prescriptive section, i.e., running through how to do scenarios (a) and (b)?